keps/sig-node/4382-dra-numeric-model-simple-counter/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: KEP Template
+kep-number: 4382
+authors:
+  - "@pohly"
+owning-sig: sig-node
+participating-sigs:
+  - sig-scheduling
+  - sig-autoscaling
+status: implementable
+creation-date: 2024-01-05
+reviewers:
+  - TBD
+  - "@alice.doe"
+approvers:
+  - TBD
+  - "@oscar.doe"
+
+see-also:
+  - "/keps/sig-node/3063-dynamic-resource-allocation"
+  - "/keps/sig-node/4381-dra-numeric-parameters"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.30"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.30"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: DRACounterNumericModel
```

Comment:
Question: Are feature gate names basic strings? What about `DRANumericModel.Counter` or even `DRANumericModel/Counter` to indicate the heirarchy?

Here I assume that this depends on `DRANumbicModel` being enabled as well, or can these be toggled individually?

Reply:
I think they are supposed to be plain strings. I'm not sure whether it's enforced anywhere, but none of the existing feature names have punctuation.

The `DRANumericParameters` feature must be enabled for `DRACounterNumericModel` to have an effect.